packages/lodestar/src/network/peers/metastore.ts

```diff
@@ -1,4 +1,4 @@
-import MetadataBook from "libp2p/src/peer-store/metadata-book";
+import {MetadataBook} from "libp2p/src/peer-store/types";
```

Comment:
Safe to import from `/src`?

Reply:
libp2p bundles `src` in its npm package and we've used it for a while so I think it's safe, @wemeetagain could you confirm

Reply:
Yeah its safe